lib/active_record/postgresql_analyzer.rb

```diff
@@ -15,11 +15,13 @@ def sql(event)
 
         return if ignore_payload?(payload)
 
+        connection = ObjectSpace._id2ref(payload[:connection_id])
```

Comment:
I'm a little worried about exception. Fundamentally `ObjectSpace._id2ref` does not raise `RangeError` in this callback. Is my understanding correct?

Reply:
> Fundamentally ObjectSpace._id2ref does not raise RangeError in this callback. 

I also understand so.
Because `payload[:connection_id]` is `connection.__id__` .

Reply:
OK! Thank you!